CustomerIOMessagingInApp.podspec

```diff
@@ -17,8 +17,12 @@ Pod::Spec.new do |spec|
   # spec.osx.deployment_target = "10.15"
   # spec.tvos.deployment_target = '13.0'
 
-  spec.source_files  = "Sources/MessagingInApp/**/*"
-  spec.exclude_files = "Sources/**/*{.md}"
```

Comment:
Is there a reason to remove `exclude_files` ? Is it something specific to version or something that I am missing out?

Reply:
Because I updated the syntax of `spec.source_files` to only include `.swift` files, `exclude_files` is no longer needed. I think using `source_files` in this way is better for long-term maintainability compared to having to keep `exclude_files` updated.

Reply:
Sound good. Can you please add this as comment for future reference?